server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -250,17 +247,19 @@ private CompletionStage<Result> submitInternal(
                 if (cause instanceof ApplicationSubmissionException) {
                   Call reviewPage =
                       routes.ApplicantProgramReviewController.review(applicantId, programId);
+                  Messages msgProvider = messagesApi.preferred(request);
                   String errorMsg =
-                      messagesApi
-                          .preferred(request)
-                          .at(MessageKey.BANNER_ERROR_SAVING_APPLICATION.getKeyName());
+                      msgProvider.at(
```

Comment:
This is other key change of this PR and is an example of how localized error messages will be handled.